### ISSUE TITLE
URL's from Redis To Go don't work due to "too many colons in address"

### ISSUE THREAD

**Issue by drauschenbach** [motiv-labs/janus#86]
A conventional Redis-To-Go URL doesn't work.

```
2017-02-26T20:54:23.538797721Z","@version":"1","level":"debug","message":"Trying to connect to mongodb://user:password@hidden.mlab.com:12345/janus","type":"Janus"}
2017-02-26T20:54:25.293989182Z {"@timestamp":"2017-02-26T20:54:25.293716857Z","@version":"1","level":"debug","message":"Connected to mongodb","type":"Janus"}
2017-02-26T20:54:25.294027164Z {"@timestamp":"2017-02-26T20:54:25.293791179Z","@version":"1","level":"debug","message":"Trying to connect to redis pool: redis://user:password@hidden.redistogo.com:12345","type":"Janus"}
2017-02-26T20:54:25.294033972Z {"@timestamp":"2017-02-26T20:54:25.293817677Z","@version":"1","level":"fatal","message":"Couldn't connect to the redis pool: dial tcp: too many colons in address redis://user:password@hidden.redistogo.com:12345","type":"Janus"}
```

**Janus version:** quay.io/hellofresh/janus:1.7.4

**Comment by italolelis** [comment]
Thank you for reporting that, the problem was that we were using the `Dial` method from the redigo library, we changed to use `DialURL` so it should work properly now.

**Comment by drauschenbach** [comment]
Confirmed [here](https://github.com/Megalithic-LLC/janus-dockerfile) that this is fixed in [1.7.4-rc.6](https://github.com/hellofresh/janus/releases/tag/1.7.4-rc.6). Thanks!
